### ISSUE TITLE
ingest service status visibility

### ISSUE THREAD

**Issue by steinbro** [soundscape-community/soundscape#71]
The tile service exposes a few methods to check its health (as noted in #67), but to my knowledge the only way to see how the ingest service is doing is to ssh into the server and look at the container logs. It would be nice to see when the last data import ran, what its status was, and how long it took. I'm open to suggestions as to how to do that.

**Comment by RDMurray** [comment]
Yes and those logs are 99% useless information. They could definitely be cleaned up. I was looking at what the Scottish Tech Army did with [the backend](https://github.com/Scottish-Tech-Army/soundscape-backend). I like that they separated the ingest service into initial import and update, rather than having a long running process. This should be easier to manage and we could schedule the updates at less busy times to avoid performance issues if necessary. A single container should be sufficient though, the scripts can be started with docker compose run.

**Comment by RDMurray** [comment]
The output from running the update script could be emailed, added to a web accessible directory, and when we decide on a monitoring service sent there.

**Comment by steinbro** [comment]
A good first step here would be to change the `telemetry_log` function from effectively a no-op to do something useful, like send the event to Sentry. The code is already instrumented with a few calls to this in key places. From there, I believe we could set up some sort of alert in Sentry that would notify us if we haven't seen any events in more than a week.

**Comment by JJGatchalian** [comment]
Replying to the comment from RDMurray, if we do something like how Scottish Tech Army ingests their data, I hope that these improvements won't create similar problems like in Scottish Tech Army. While I was adding places to Open Street Map in my own city last year, I was experimenting with the two versions of Soundscape, and here's what I found. I prefer Soundscape Community's ingestion process where the entire map gets updated every couple of weeks. It's nice and simple, and everything gets updated all at once. IOS Scottish Tech Army updates nodes every two minutes, but the ways never get updated. Also, if I delete something from Open Street Map that no longer exists, IOS Scottish Tech Army doesn't delete it in the first place. That means that some places marked as ways never got updated since the service launched, and anything that got deleted, node or way, still shows up also. That's why I still have Scottish Tech Army on my phone even though Soundscape Community is my go-to version for everyday use. I use Scottish Tech Army to keep track of the places I added to Open Street Map because they'll show up in just a few minutes. As for Soundscape Community, we have to ensure that the ingestion service improvements handle every case so that we can get the entire map updated as it's doing currently. I believe that the Android Scottish Tech Army version under development is using a totally different service that's beyond this scope.

**Comment by steinbro** [comment]
> I prefer Soundscape Community's ingestion process where the entire map gets updated every couple of weeks. It's nice and simple, and everything gets updated all at once. IOS Scottish Tech Army updates nodes every two minutes, but the ways never get updated. Also, if I delete something from Open Street Map that no longer exists, IOS Scottish Tech Army doesn't delete it in the first place. That means that some places marked as ways never got updated since the service launched, and anything that got deleted, node or way, still shows up also.

Yeah, I wouldn't be inclined to make any major changes here like importing incremental updates. As you observed, that can introduce bugs where the database slowly drifts out of sync with the published data. The goal of this ticket is just to add some observability to the existing process.
